composeApp/src/commonMain/kotlin/sp/bvantur/tasky/home/ui/HomeScreen.kt

```diff
@@ -117,3 +144,35 @@ fun FloatingActionButtonWithDropdown(onCreateEventAction: () -> Unit) {
         }
     }
 }
+
+@Composable
+private fun ProfileInitials(modifier: Modifier = Modifier, name: String, onLogoutAction: () -> Unit) {
+    var showMenu by remember { mutableStateOf(false) }
```

Comment:
Your dropdown will collapse when a configuration changes (e.g., the device orientation changes or the user toggles between light and dark mode). I suggest you save this state in your VM or use `rememberSaveable`, both of which survive configuration changes. I always lean towards using the VM for state management to keep my UI as dumb as possible and let it focus purely on rendering the state provided to it.